design/ingressroute-design.md

```diff
@@ -353,23 +353,19 @@ _TBD_
 
 Metrics are essential to any system. Contour will expose a `/metrics` Prometheus endpoint with the following metrics:
 
-- **contour_ingressroute_total (gauge):** Number of IngressRoutes
+- **contour_ingressroute_total (gauge):** Total number of IngressRoutes objects that exist regardless of status (i.e. Valid / Invalid / Orphaned, etc). This metric should match the sum of `Orphaned` + `Valid` + `Invalid` IngressRoutes.
   - namespace
   - vhost
-- **contour_ingressroute_upstream_total (gauge):** Number of Upstreams per IngressRoute
+- **contour_ingressroute_orphaned_total (gauge):**  Number of `Orphaned` IngressRoute objects which have no root delegating to them
   - namespace
-  - name
-- **contour_ingressroute_upstream_endpoints_total (gauge):** Number of Endpoints per Upstream
+- **contour_ingressroute_root_total (gauge):**  Number of `Root` IngressRoute objects (Note: There will only be a single `Root` IngressRoute per vhost)
```

Comment:
Would it be useful to capture the VHost here?

Reply:
It will only ever be `1`, so not sure if knowing that root is there is what we're after or how many roots are in the namespace. @rosskukulinski what do you think?

Reply:
I don't think it would be helpful to capture vhost for contour_ingressroute_root_total.